src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +89,32 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    /**
+     * @return the blockNumber
+     */
+    public Block getBlockNumber() {
+        return blockNumber;
+    }
+
+    /**
+     * @return the streetName
+     */
+    public Street getStreetName() {
+        return streetName;
+    }
+
+    /**
+     * @return the unitNumber
+     */
+    public Unit getUnitNumber() {
+        return unitNumber;
+    }
+
+    /**
+     * @return the postalCode
```

Comment:
For getter methods, header comments are optional. But if you do put them add a meaningful header comment. Right now it does not add any value to the code.